src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -149,32 +149,33 @@ static LinkedFile convert(List<String> entry) {
         LinkedFile field = null;
         if (LinkedFile.isOnlineLink(entry.get(1))) {
             try {
-                field = new LinkedFile(entry.getFirst(), new URL(entry.get(1)), entry.get(2));
```

Comment:
Sure, will revert that change, it feels very counter-intuitive the method difference in the same line but if it's somehow standard...

Reply:
Yes, that could be argued and I agree to some extend.

We are using rewriter to keep some basic code quality within the contributions of cs students who are doing some of their first steps in open source programming with jabref. I guess you could somehow create an exception for the rewriter, but that would probably exceed the limits of this pr...

Reply:
Ah no definitely, I made that change because I was looking at the code and found this inconsistency, but if the code linter you use prefers it that way that's just how it is, not a problem with me.